packages/page-contracts/src/Contracts/Call.tsx

```diff
@@ -35,10 +36,10 @@ function Call (props: Props): React.ReactElement<Props> | null {
   const hasRpc = callContract?.hasRpcContractsCall;
   const callMessage = callContract?.getMessage(isNull(callMessageIndex) ? undefined : callMessageIndex);
 
-  const [accountId, setAccountId] = useState<StringOrNull>(null);
-  const [endowment, setEndowment] = useState<BN>(new BN(0));
-  const [gasLimit, setGasLimit] = useState<BN>(new BN(DEFAULT_GAS_LIMIT));
-  const [isBusy, setIsBusy] = useState(false);
+  const [accountId, setAccountId] = useAccountId();
+  const [endowment, isEndowmentValid, setEndowment] = useFormField<BN>(new BN(0));
+  const [gasLimit, isGasLimitValid, setGasLimit] = useNonZeroBn(new BN(GAS_LIMIT));
+  const [isBusy, , setIsBusy] = useToggle();
```

Comment:
```suggestion
  const [isBusy, setIsBusy] = useToggle();
```

Reply:
This is not quite correct - `[isBusy, toggleIsBusy, setIsBusy]`, so the usage here is correct.

Reply:
Ah sorry, my bad

Reply:
No issues, extra eyeballs are _always_ good.